### PR TITLE
Additional renovate PR labels

### PR DESCRIPTION
**Pull request by ab77**
https://github.com/product-os/policies/pull/18

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -20,7 +20,12 @@
   },
   "labels": [
     "renovate",
-    "dependencies"
+    "dependencies",
+    "isPatch:{{isPatch}}",
+    "isMinor:{{isMinor}}",
+    "isMajor:{{isMajor}}",
```
Comment (klutchell) [inline .github/renovate.json:26]: These booleans seem like overkill. Is there any reason we can't just use `updateType:` from below?

Comment (klutchell) [reply .github/renovate.json:26]: I mean they are fine, it just seems like duplication of logic and noise.

Comment (ab77) [reply .github/renovate.json:26]: Fair point, from [here](https://docs.renovatebot.com/configuration-options/#matchupdatetypes), `updateType` will evaluate to:

```
* major
* minor
* pin
* pin
* digest
```

File: .github/renovate.json [hunk 2]
```diff
@@ -20,7 +20,8 @@
   },
   "labels": [
     "renovate",
-    "dependencies"
+    "dependencies",
+    "{{updateType}}"
```
Comment (klutchell) [inline .github/renovate.json:24]: including `depType` wasn't a terrible idea though... :D

**Comment by klutchell** [comment]
Squash commits, then lgtm

**Comment by ab77** [comment]
lgtm

**Pull request merged**